DIS.html

```diff
@@ -43,23 +43,16 @@ <h3>Section One - Using the DIS</h3>
         
     </p>
     <ol>
-        <li>Open <a href="https://jenkins.nelthorya.net/job/DSIHaxInjector/">this website</a></li>
-        <li>Click <code>Build with Parameters</code> on the left side</li>
-        <li>Enter your name in the Username Field - <strong>Any name will work</strong>, just dont make it too short or
-            include spaces.
-        </li>
+        <li>Open <a href="https://jenkins.nelthorya.net/job/DSIHaxInjector/build">this website</a></li>

```

Comment:
Please readd the note about the name, i have seen multiple people fail because of it and asking again "what username?!"

Reply:
Think we just left that step out by accident while we were trying to condense it tbh. (even though we know it's stupid we need to tell people what to do that's already on the page, yes we've seen it happen as well)